lib/components/viewers/trip-viewer.js

```diff
@@ -25,19 +25,33 @@ import ViewStopButton from './view-stop-button'
 const { getCurrentDate } = coreUtils.time
 
 const StopList = styled.ol`
+  list-style: none;
   padding-left: 0;
 `
 const Stop = styled.li`
-  list-style: none;
+  align-items: center;
+  display: flex;
 `
 const RouteName = styled.h2`
-  margin-bottom: 20px;
   font-size: inherit;
+  margin: 0 0 1em 0;
 `
 const HeaderText = styled.h1`
   margin: 2px 0 0 0;
 `
 
+const TripDescription = styled.div`
```

Comment:
Or use `<Alert bsStyle="info">` (as long as we are using bootstrap)

Reply:
Agree it looks very similar to Bootstrap, are we trying to move away from Bootstrap long term?

Reply:
Agreed I think we should avoid adding any more bootstrap

Reply:
But perhaps making this `TripDescription` thing into a custom `Alert` component and putting in utils might be a good middle ground!

Reply:
added a component to utils in fb0e0f4b1006719b8ae4fbf1f1e3e3f80d95c2e7 !